src/payload/payload.config.ts

```diff
@@ -19,7 +19,7 @@ import { Posts } from "./collections/Posts";
 import { Projects } from "./collections/Projects";
 import { Contributors } from "./collections/Contributors";
 import Users from "./collections/Users";
-import { PodcastEpisodes } from "./collections/PodcastEpisodes";
+import { PodcastEpisodes } from "./collections/PodcasstEpisodes";
```

Comment:
`PodcasstEpisodes`? 😂

Reply:
Inspired by "snake jazz" 😆 👉 👈 

i think it's fixed on the the next ones

Reply:
fixed ✅